scripts/eventEntry.js

```diff
@@ -1,27 +1,42 @@
 (function(module) {
   var firebasedb = firebase.database()
 
-  var eventHandler = {};
-  eventHandler.save = function (e) {
+  var newEventHandler = {};
+
+  //submit a new event
+  newEventHandler.save = function (e) {
     e.preventDefault();
-    var newEvent = new Event( $('#save-event input').get().reduce(function(newObj, cur){
-      newObj[cur.id] = $(cur).val();
-      return newObj;
-    }, {})
-  );
-    Event.getLatandLog(newEvent, newEvent.address);
+    // check if the date is in the future
+    var date = new Date($("#date").val());
+    if (date - new Date < 0) {
+      $('#date-alert').removeClass('hidden')
```

Comment:
Small Jquery optimization: if you don't expect the `$('#date-alert')` element to change, you can do something like `var $dateAlert = $('#date-alert');` and then do manipulations on the variable. This saves JQuery from doing a fresh lookup every time.